int32 lives
int32 score
int32 gtime
float32 performEval
